src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexCharClass.cs

```diff
@@ -1344,7 +1343,7 @@ internal LowerCaseMapping(char chMin, char chMax, int lcOp, int data)
         /// </summary>
         private sealed class SingleRangeComparer : IComparer<SingleRange>
         {
-            public static readonly SingleRangeComparer Instance = new SingleRangeComparer();
+            public static readonly SingleRangeComparer s_instance = new SingleRangeComparer();
```

Comment:
Nit: This is a public field, so we generally have used PascalCasing.  It just happens to be on a private type.

Reply:
Changing to internal would be non breaking?

Reply:
On a private class